packages/rancher-prom-to-teams/generated-changes/exclude/templates/tests/test-connection.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  name: "{{ include "prom2teams.fullname" . }}-test-connection"
+  labels:
+{{ include "prom2teams.labels" . | indent 4 }}
```

Comment:
Can't quite tell, does this include the tolerations/selectors ?

Reply:
Or are we actually removing this ? Can't quite tell what is going on here.

Reply:
this file is excluded from the final chart.